openedx/features/genplus_features/genplus_learning/api/v1/views.py

```diff
@@ -41,3 +42,30 @@ def get_permissions(self):
         else:
             permission_classes.append(IsTeacher)
         return [permission() for permission in permission_classes]
+
+
+class ClassStudentViewSet(mixins.ListModelMixin,
+                          viewsets.GenericViewSet):
+    authentication_classes = [SessionAuthenticationCrossDomainCsrf]
+    permission_classes = [IsAuthenticated, IsStudentOrTeacher]
+    serializer_class = ClassStudentSerializer
+    filter_backends = [filters.SearchFilter]
+    search_fields = ['gen_user__user__username']
+
+    def get_serializer_context(self):
+        context = super(ClassStudentViewSet, self).get_serializer_context()
+        group_id = self.kwargs.get('group_id', None)
+        if group_id:
+            gen_class = get_object_or_404(Class, group_id=group_id)
+            class_units = ClassUnit.objects.select_related('unit').prefetch_related('class_lessons')
```

Comment:
can we remove this prefetch class_lessons if we are not using this anymore?

Reply:
Sure